### PR TITLE
move cookies and cgi to graveyard

### LINKED ISSUE #7810: Remove cookies module from stdlib
This module really shouldn't be in the stdlib in the first place IMO

### PR DESCRIPTION
**Pull request by narimiran**
This supersedes PR #9360, and closes #7810.

### CODE REVIEW

**Comment by narimiran** [comment]
Unrelated appveyor fail. Don't want to re-run everything since our queue is getting too large and too slow.

File: lib/pure/uri.nim [hunk 1]
```diff
@@ -463,6 +471,107 @@ proc `$`*(u: Uri): string =
     result.add("#")
     result.add(u.anchor)
 
+
+proc cgiError*(msg: string) {.noreturn.} =
```
Comment (Araq) [inline lib/pure/uri.nim]: Gah, that's terrible.

**Comment by Araq** [comment]
I don't like it at all, we should keep these modules in the stdlib if they are used and reasonably free of issues. They don't cost us much maintainance.

**Comment by narimiran** [comment]
Ok, then I'm closing #7810 and putting "won't fix" label there.

**Pull request closed without merging**